doc_classes/KhronosEditorExportPlugin.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="KhronosEditorExportPlugin" inherits="OpenXREditorExportPlugin" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
```

Comment:
@akien-mga left a comment on the godot-cpp PR about the invalid link to `class.xsd`.